modules/2_nbde/nbde.tf

```diff
@@ -125,6 +147,16 @@ EOF
     ]
   }
 
+  provisioner "remote-exec" {
+    when = create
+    inline = [
+      <<EOF
+ansible-galaxy collection install community.general:6.4.0 ansible.posix
```

Comment:
Can you double check ... 

I expect it to be 

```
ansible-galaxy collection install community.general:6.4.0
ansible-galaxy collection install ansible.posix:1.5.1
```

fedora.linux_system_roles:==1.33.0

Reply:
Yes have added this while validating code but missed in updating PR.  Added change - Thanks